src/test/java/com/example/animalshelter/controller/AnimalControllerIT.java

```diff
@@ -31,27 +42,44 @@ public class AnimalControllerIT {
     @Autowired
     private MockMvc mockMvc;
 
-    @Test
-    public void contextLoad() {
-        assertThat(animalController).isNotNull();
+    private final Animal dummyAnimal = createAnimal();
+    private final Animal secondDummyAnimal = createSecondAnimal();
+
+    private Animal createAnimal() {
```

Comment:
please move all private methods to the end of the test file or move them to the Fixture class.